buildSrc/subprojects/configuration/src/main/kotlin/org/gradle/gradlebuild/java/AvailableJavaInstallations.kt

```diff
@@ -27,18 +28,26 @@ class DefaultJavaInstallation(val current: Boolean, private val javaHome: File)
     override fun setJavaHome(javaHome: File) { throw UnsupportedOperationException("JavaHome cannot be changed") }
     val toolsJar: File? by lazy { Jvm.forHome(javaHome).toolsJar }
 
-    override fun toString(): String = "${displayName} (${javaHome.absolutePath})"
+    override fun toString(): String = "$displayName (${javaHome.absolutePath})"
 }
 
+
 private
 const val java7HomePropertyName = "java7Home"
+
+
 private
 const val testJavaHomePropertyName = "testJavaHome"
+
+
```

Comment:
Why two newlines between all those declarations? Is this somehow Kotlin std?

Reply:
It is the `gradle/kotlin-dsl` convention, see https://github.com/gradle/kotlin-dsl/blob/5f1740de6e3129f1b37120243d9afe682405fe77/plugins-experiments/src/test/kotlin/org/gradle/kotlin/dsl/experiments/plugins/GradleKotlinDslKtlintConventionPluginTest.kt#L115-L179

This, like other rules enforced by this convention, is what we've been doing in `gradle/kotlin-dsl`.

Reply:
Also see https://github.com/gradle/kotlin-dsl/pull/750 for more info

Reply:
So top level declarations should be separated by two lines.
I still don't understand why we are doing this...
Probably mostly for not having anything similar in Java/Groovy/Checkstyle

Reply:
@wolfs it's our experience with the `kotlin-dsl` codebase that this makes for more readable sources as Kotlin files tend to have many top-level declarations.